package.json

```diff
@@ -151,6 +151,7 @@
     "@types/lerna__project": "5.1.0",
     "@types/loader-utils": "2.0.3",
     "@types/markdown-table": "2.0.0",
+    "@types/micromatch": "4.0.2",
```

Comment:
Added to fix this error in scripts:

![image](https://user-images.githubusercontent.com/8649804/195950618-c3eafaa8-610d-400a-95ee-c668dba6da12.png)